src/pages/components/externalPages/ExternalPageTemplate.js

```diff
@@ -22,6 +23,7 @@ const ExternalPageTemplate = ({ title, children }) => (
     </main>
     <FooterContent />
     <CookiePolicy />
+    <ContactUs />
```

Comment:
```suggestion
    <Chatbot colorScheme={colorScheme} />
```

Rename the component to Chatbot and pass the colorScheme props and make sure to import colorScheme and useTheme context